docs/fetch_tutorial.rst

```diff
@@ -1548,8 +1548,10 @@ local or remote.
 In order to use this option, the user must have a special key file
 ``ska_remote_access.json`` placed at the root of the local Python installation
 folder.  This is placed in the directory shown with ``import sys;
-print(sys.prefix)``.  To get a copy of this file contact Mark Baski or Tom
-Aldcroft.
+print(sys.prefix)``.
+
+To get a copy of this file contact Mark Baski or Tom Aldcroft, or look in the
```

Comment:
This isn't a URL, but does give a hint about finding this key file. Given that VPN network access is needed to actually do anything with this, seems fine to me but I could leave this out if people are worried.